README.md

```diff
@@ -1,6 +1,4 @@
-# Getting Started with Create React App
-
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Getting Started with React Rampup
```

Comment:
Add all the steps one has to do to run the project.

Reply:
Added